src/mono/wasm/build/WasmApp.targets

```diff
@@ -256,8 +258,10 @@
       <_PythonToolsBasePath>$(EmscriptenSdkToolsPath)python</_PythonToolsBasePath>
       <_PythonToolsVersionedPath Condition="Exists($(_PythonToolsBasePath))">$([System.IO.Directory]::GetDirectories($(_PythonToolsBasePath)))</_PythonToolsVersionedPath>
       <EmscriptenPythonToolsPath Condition="'$(_PythonToolsVersionedPath)' != ''">$(_PythonToolsVersionedPath)</EmscriptenPythonToolsPath>
+      <EmscriptenPythonBinPath   Condition="'$(EmscriptenPythonToolsPath)' != ''">$(EmscriptenPythonToolsPath)\bin\</EmscriptenPythonBinPath>
```

Comment:
Is this correct?

Reply:
It is, when using emsdk checkout, like `src/mono/wasm/emsdk/python/3.7.4-2_64bit/bin`

Reply:
on windows too?

Reply:
on windows I have `src\mono\wasm\emsdk\python\3.7.4-pywin32_64bit\python.exe`